[2023-05-18T06:45:30.725+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data manual__2023-05-18T06:45:01.052161+00:00 [queued]>
[2023-05-18T06:45:30.753+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data manual__2023-05-18T06:45:01.052161+00:00 [queued]>
[2023-05-18T06:45:30.756+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-18T06:45:30.797+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-18 06:45:01.052161+00:00
[2023-05-18T06:45:30.812+0000] {standard_task_runner.py:57} INFO - Started process 2808 to run task
[2023-05-18T06:45:30.816+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'manual__2023-05-18T06:45:01.052161+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpuj2zj1hs']
[2023-05-18T06:45:30.820+0000] {standard_task_runner.py:85} INFO - Job 45: Subtask clean_fondue_data
[2023-05-18T06:45:30.933+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data manual__2023-05-18T06:45:01.052161+00:00 [running]> on host 714a0c7ad307
[2023-05-18T06:45:31.185+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-18T06:45:01.052161+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-05-18T06:45:01.052161+00:00'
[2023-05-18T06:45:31.402+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-18T06:45:31.502+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-18T06:45:31.530+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230518T064501, start_date=20230518T064530, end_date=20230518T064531
[2023-05-18T06:45:31.595+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-18T06:45:31.672+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
